nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_width.py

```diff
@@ -758,6 +797,18 @@ def get_group_id_by_node_name(self, node_name: NNCFNodeName) -> Optional[int]:
                     group_id = cluster.id
         return group_id
 
+    def get_importance_by_node_name(self, node_name: NNCFNodeName):
+        """
+        Return custom weight importance for the current node.
+
+        :param node_name: node name
+        :return: importance tensor
+        """
+        # Map node name to module name
+        key = ".".join(re.findall(re.compile(r"[[](.*?)[]]", re.S), node_name)) + ".weight"
```

Comment:
This code does not look general. Node name is defined on the building NNCFGraph.
please use the dedicated matching functions for node name - https://github.com/openvinotoolkit/nncf/blob/c732de08a14ce389a3a435fc70d53ee9ebd91564/nncf/common/scopes.py#L52

Reply:
@ljaljushkin  Fixed. Thanks!